src/ui/components/Notice/index.js

```diff
@@ -58,6 +58,7 @@ type Props = {|
   dismissible?: boolean,
   id?: string,
   light?: boolean,
+  multiline?: boolean,
```

Comment:
Do you think we could infer the value of `multiline` by counting the number of `children`? We could probably use something like this: https://reactjs.org/docs/react-api.html#reactchildrencount

The problem with a prop like this one is that it's easy to forget. Maybe we don't need an public prop. `render()` could declare a `multiline` variable assigned to the result of `React.Children.count()`.

WDYT?